src/cryptoadvance/specter/server_endpoints/filters.py

```diff
@@ -59,12 +59,17 @@ def btcunitamount(context, value):
 @filters_bp.app_template_filter("altunit")
 def altunit(context, value):
     if app.specter.price_check and (app.specter.alt_rate and app.specter.alt_symbol):
-        return (
-            "{:,.2f}".format(float(value) * float(app.specter.alt_rate))
-            .rstrip("0")
-            .rstrip(".")
-            + app.specter.alt_symbol
-        )
+        if app.specter.alt_symbol == "€":
+            return (
+                "{:,.2f}".format(float(value) * float(app.specter.alt_rate))
+                .rstrip("0")
+                .rstrip(".")
+                + app.specter.alt_symbol
+            )
+        else:
+            return app.specter.alt_symbol + "{:,.2f}".format(
+                float(value) * float(app.specter.alt_rate)
+            ).rstrip("0").rstrip(".")
```

Comment:
```suggestion
       rate = (
            "{:,.2f}".format(float(value) * float(app.specter.alt_rate))
            .rstrip("0")
            .rstrip(".")
        )
        if app.specter.alt_symbol in ["$", "£"]:
            return app.specter.alt_symbol + rate
        else:
            return rate + app.specter.alt_symbol
```
I'd suggest doing it like this. Both cleaner and because I think the default should be to the right, since if you use the manual price settings for stuff which don't have symbol (So you put usually something like ticker symbol, I think they should be on the right. (I.e. 1 USD, 10 AUX etc.). Does that make sense?

Reply:
Agreed, I did not think about that! Makes sense.